invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -94,20 +94,36 @@ def get_affiliation(self, obj):
                 }
                 identifiers = affiliation.get("identifiers")
                 if identifiers:
-                    # PIDS-FIXME: DataCite accepts only one, how to decide
-                    identifier = identifiers[0]
+                    # FIXME: Make configurable
+                    DATACITE_AFFILIATION_IDENTIFIER_TYPES_PREFERENCE = (
```

Comment:
Does this make sense to be configurable? Does this list get updated? If not, I would remove the FIXME otherwise I would export this to the module's config. What do you think @slint @tmorrell ?

Reply:
This should be configurable, in the sense that it decides what is the preferred identifier for sending to DataCite, which is an instance preference at the end (since there's no recommendation from the DataCite schema on what is "the" canonical identifier type for various entities like authors, orgs, funders, etc.).

I think the difficulty with extracting into config at the moment, might be that we do unit testing without assuming an application context pushed, so it might turn into a much bigger task of refactoring tests.

Reply:
I agree with @slint that this would be good to make configurable in the future, but the complexity of modifying the config and tests is too much for the moment. We haven't had anyone request to have this configuration, and if we don't merge this for v11 we're going to continue to be sending missing metadata to DataCite.